### DISCUSSION TITLE
Time To Live (TTL)

### DISCUSSION THREAD

**Discussion by tekoscott** [jchristn/SimpleUdp#9] [General]
Does this library support setting the TTL on each packet?

Kind Regards

**Comment by jchristn** [comment]
Hi @tekoscott sorry, I never received a notification from Github on this issue!  Currently the library does not.  Happy to accept a PR on that if you're interested.

**Comment by tekoscott** [comment]
No problems, i can do a pull request tonight after work.

**Comment by jchristn** [comment]
Working on one for you now

**Comment by jchristn** [comment]
Please let me know if this works for you.  TTL is now an optional param at the end of each send call.  Default is 64, minimum is 0.

NuGet v1.1.0.1: https://www.nuget.org/packages/SimpleUdp/1.1.0.1
Commit: https://github.com/jchristn/SimpleUdp/commit/4d523e2a8da364e647e0da1eac0b9f57366b2d75

Cheers, Joel

**Reply by tekoscott** [reply]
Great! Works like a charm thanks for that
